src/library/Uno.Material/Styles/Controls/v2/ProgressBar.xaml

```diff
@@ -5,19 +5,38 @@
 					xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
 					mc:Ignorable="d">
 
-	<x:Double x:Key="MaterialProgressBarMinWidth">250</x:Double>
-	<x:Double x:Key="MaterialProgressBarHeight">4</x:Double>
-	
+	<x:Double x:Key="ProgressBarMinWidth">250</x:Double>
```

Comment:
Let's keep the `ThemeDictionaries` declaration at the top of the file and leave non theme resources to be declared afterwards